documentation/asciidoc/computers/ai/getting-started.adoc

```diff
@@ -154,8 +151,40 @@ This demo performs 17-point human pose estimation, drawing lines connecting the
 
 [source,console]
 ----
-$ rpicam-hello -t 0 --post-process-file ~/rpicam-apps/assets/hailo_yolov8_pose.json --lores-width 640 --lores-height 640
+$ rpicam-hello -t 0 --post-process-file /usr/share/rpi-camera-assets/hailo_yolov8_pose.json
+----
+
+=== Alternative Package Versions
+
+The AI Kit/HAT+ devices will not function if there is a version mismatch between the Hailo software packages and device drivers. In addition, Hailo's neural network tooling may also require particular a version for the generated model file. At the time of writing, the latest package version is 4.19, but if earlier versions are required, the following steps should be followed:
+
+. If you have previously used `apt-mark` to hold any of the relevant packages, you may need to unhold them:
++
+[source,console]
+----
+$ sudo apt-mark unhold hailo-tappas-core hailort hailo-dkms
+----
+
+. Install the required version of the software packages
++
+[tabs%sync]
+====
+v4.18::
++
+[source,console]
+----
+$ sudo apt install hailo-tappas-core=3.29.1 hailort=4.18.0 hailo-dkms=4.18.0-2
+$ sudo apt-mark hold hailo-tappas-core hailort hailo-dkms
+----
+
+v4.17::
++
+[source,console]
+----
+$ sudo apt install hailo-tappas-core-3.28.2 hailort=4.17.0 hailo-dkms=4.17.0-1
+$ sudo apt-mark hold hailo-tappas-core hailort hailo-dkms
 ----
+====
```

Comment:
```suggestion
======
v4.18::
To install version 4.18 of Hailo's neural network tooling:
+
+
[source,console]
----
$ sudo apt install hailo-tappas-core=3.29.1 hailort=4.18.0 hailo-dkms=4.18.0-2
----
+
[source,console]
----
$ sudo apt-mark hold hailo-tappas-core hailort hailo-dkms
----

v4.17::
To install version 4.17 of Hailo's neural network tooling:
+
[source,console]
----
$ sudo apt install hailo-tappas-core-3.28.2 hailort=4.17.0 hailo-dkms=4.17.0-1
----
+
[source,console]
----
$ sudo apt-mark hold hailo-tappas-core hailort hailo-dkms
----
======
```

Reply:
You wouldn't know it without testing, but console snippets are awkward to copy/paste when we smush them together into a single code block. So I tend to split them up!